app/src/pages/admin/projectEventsPage/projectEventsPage.jsx

```diff
@@ -68,9 +71,11 @@ export class ProjectEventsPage extends Component {
       onChangePage,
       onChangePageSize,
       loading,
+      events,
     } = this.props;
     return (
       <React.Fragment>
+        <EventsGrid data={events} loading={loading} />
         {!!pageCount &&
           !loading && (
             <PaginationToolbar
```

Comment:
Please, also fix the pagination issues (see the screenshot below).
![image](https://user-images.githubusercontent.com/31861805/51765973-66c2f680-20ea-11e9-9a0d-2b1d5e6ba506.png)

Reply:
We have a separate task for this bug, so I'll try to fix it there